src/Sinch/Voice/Callouts/Callout/TextToSpeechCalloutRequest.cs

```diff
@@ -76,7 +76,7 @@ public class TtsCalloutRequest
         ///     Example:
         ///     <c>#tts[Hello from Sinch];#ssml[&lt;speak&gt;&lt;break time=\"250ms\"/&gt;Have a great day!&lt;/speak&gt;]</c>
         /// </summary>
-        public string Promts { get; set; }
+        public string? Promts { get; set; }
```

Comment:
Typo

Reply:
That will be in separate as well

Reply:
In this case, please create a ticket and communicate the reference to the reviewers